src/main/java/io/fabric8/maven/docker/service/BuildService.java

```diff
@@ -27,13 +13,15 @@
 import io.fabric8.maven.docker.config.ImageConfiguration;
 import io.fabric8.maven.docker.model.ImageArchiveManifest;
 import io.fabric8.maven.docker.model.ImageArchiveManifestEntry;
-import io.fabric8.maven.docker.util.DockerFileUtil;
-import io.fabric8.maven.docker.util.EnvUtil;
-import io.fabric8.maven.docker.util.ImageArchiveUtil;
-import io.fabric8.maven.docker.util.ImageName;
-import io.fabric8.maven.docker.util.Logger;
-import io.fabric8.maven.docker.util.MojoParameters;
-import io.fabric8.maven.docker.util.NamePatternUtil;
+import io.fabric8.maven.docker.util.*;
+import org.apache.maven.plugin.MojoExecutionException;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.Serializable;
+import java.nio.file.Files;
+import java.util.*;
```

Comment:
nit, Could you please avoid using wildcard imports?

Reply:
done :+1: (sorry I re-installed IntelliJ and forgot to change this setting)